index.html

```diff
@@ -138,8 +156,11 @@ <h5>Languages</h5><img src="./images/Union.png" alt="expand" class="arrow"></li>
               <li class="languages"><img src="./images/Ellipse2.png" alt="html logo"><h6>HTML</h6></li>            
               <li class="languages"><img src="./images/Ellipse3.png" alt="css logo"><h6>CSS</h6></li>
             </ul></li>
+          <li class="separator"><img src="./images/Separator-Bottom.svg " alt="twitter"></li>
```

Comment:
- [ ] It is good that you have added the horizontal separator lines. However, you should adjust the left and right alignment of these lines as per the Figma design.
#### Design
![languaes-section-hr](https://user-images.githubusercontent.com/44057863/166105617-327c6a52-6014-4a36-9d46-1613f3815cf7.png)
.
#### Implementatio
![languaes-section-hr-impl](https://user-images.githubusercontent.com/44057863/166105689-e4b22df0-1f8a-4a2a-9172-aadea02a23ef.png)

- [ Optional ] You may also use `<hr>` tag instead of images. Please take a look at this [example](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_hr_test).

Reply:
Thanks a lot. I did not know about `<hr>`.